frappe/desk/search.py

```diff
@@ -117,18 +115,24 @@ def search_widget(
 
 	meta = frappe.get_meta(doctype)
 
-	if isinstance(filters, dict):
+	if filters is None:
+		filters = []
+	elif isinstance(filters, dict):
 		filters_items = filters.items()
 		filters = []
 		for key, value in filters_items:
 			filters.append(make_filter_tuple(doctype, key, value))
 
-	if filters is None:
-		filters = []
 	or_filters = []
 
 	# build from doctype
-	if txt:
+	search_fields = ["name"]
+	if meta.title_field:
+		search_fields.append(meta.title_field)
+	if meta.search_fields:
+		search_fields.extend(meta.get_search_fields())
+
+	if txt and not meta.translated_doctype:
```

Comment:
Can‘t imagine the `if txt` condition being relevant for the `search_fields.append` and `.extend` lines

Combining `txt and not meta.translated_doctype` saves us a whole level of indentation here.

Let‘s look at the tests. If they‘re passing and you‘re not alarmed either, I think, this is okay.